src/ERC20DripsHub.sol

```diff
@@ -17,12 +23,15 @@ contract ERC20DripsHub is DripsHub {
     /// @param owner The initial owner of the contract with managerial abilities.
     /// @param _erc20 The address of an ERC-20 contract which tokens the drips hub will work with.
     /// The supply of the tokens must be lower than `2 ^ 127`.
+    /// @param _reserve The address of the funds reserve to be used.
     constructor(
         uint64 cycleSecs,
         address owner,
-        IERC20 _erc20
+        IERC20 _erc20,
```

Comment:
I wouldn't pass the erc20 here, it is just a way for additional potential misstakes. 

We could get it from the reserve. `reserve.erc20()` or add the check if they are the same. 

I think it makes sense to store the address of the erc20 in the hub contract to make the approval  calls cheaper

Reply:
Clever! Fixed